packages/perseus/src/widgets/interactive-graphs/movable-point.tsx

```diff
@@ -38,11 +38,10 @@ export const MovablePoint = (props: Props) => {
             }
         >
             <circle className="movable-point-hitbox" r={30} cx={x} cy={y} />
```

Comment:
**Optional suggestion**: Maybe also move the hitbox radius to the styles for consistency?